apps/remix-ide/src/app/tabs/runTab/settings.js

```diff
@@ -319,6 +323,22 @@ class SettingsUI {
     )
   }
 
+  getSelectedAccount () {
+    return this.el.querySelector('#txorigin').selectedOptions[0].value
+  }
+
+  setSelectedAccount (account) {
```

Comment:
shouldn't that require feedback to the user (like a validation step)? was that also asked by plugin developer?

Reply:
ok

Reply:
added permissions